mineral-ui/mineral-ui
# [Website] Add new tokens page

### Description

* Add new Tokens page
* Fix Choice border styles in website
* Update Theming page table to display categorized variables
* Code gen using JS objects and JSON.stringify instead of string concatenation
* Wrap color non-string literal property keys in brackets to avoid flow warnings
* Remove private file `groupedTokens`
* Generate categorized tokens files in website for theming and tokens pages
* Remove eslint exclusion from mineral-ui-tokens `src`
* Add workflow step - run “npm run format” prior to committing generated files.

### Motivation and context

Closes #665 

### Screenshots, videos, or demo, if appropriate

https://665-tokens-page--mineral-ui.netlify.com

### How to test

1. `cd mineral-ui && npm start`
2. Navigate to Tokens; verify content & that tables render properly
3. Navigate to Theming; verify tables are organized by category & verify that theme & gray ramps display correctly

### Types of changes

- Refactor mineral-ui-tokens package
- New website page
- Breaking change - Fix incorrect sass variable name format

### Checklist
